server/app/routes/v1/nodes_api.rb

```diff
@@ -52,6 +52,17 @@ def load_grid_node(grid_name, node_id)
 
             render('host_nodes/health')
           end
+
+          # GET /v1/nodes/:grid/:node/stats
+          r.on 'stats' do
+            default_seconds = 60 * 60
```

Comment:
You can use also `1.hour.to_i` which is more human readable (ActiveSupport provides that functionality). Actually you don't even need to cast it to integer `to - 1.hour` should work.